### ISSUE TITLE
[dart transformers][phase 1] dry-run the transformer on all example apps and benchmarks

### ISSUE THREAD

**Issue by yjbanov** [angular/angular#1232]
This is phase 1 of a 3-phase project to productionize Dart transformers. The project does not include any optimizations, only feature-completeness. Optimizations will be planned separately.

# Phase 1 Goals
During this phase we want to make sure the transformer is feature-complete enough to process all Angular 2 apps in angular/angular/modules/{examples,benchmarks} and:

- to not crash
- to produce Dart code with no analyzer warnings
- to run dry-run as part of our Travis CI build

# Non-goals

- it is not required that transfomer-generated code passes any tests
- it is not required that transformer can transform unit-test code

**Comment by jakemac53** [comment]
blocked on https://github.com/angular/angular/issues/2984 and https://github.com/angular/angular/issues/2919

**Comment by mhevery** [comment]
I believe this is obsolete. Closing, reopen if you feel otherwise.
